kuma/wiki/middleware.py

```diff
@@ -33,8 +33,16 @@ def process_request(self, request):
                 ('$subscribe' in request.path or '$files' in request.path)):
             return None
 
+        # Process the request path
+        path = request.path_info
+        request_slug = path.lstrip('/')
+        maybe_lang = request_slug.split(u'/')[0]
+        if maybe_lang in settings.ENABLED_LOCALES:
```

Comment:
Nit. Since the code is run for almost every request, it'd be nice to make this more efficient by using a `set` or `frozenset` that we create once and use thereafter, so perhaps adding something like `ENABLED_LOCALES_SET = frozenset(ENABLED_LOCALES)` within `kuma/settings/common.py` and then using that here?